src/volatile.rs

```diff
@@ -15,6 +21,12 @@ impl<T: Copy> ReadOnly<T> {
 #[repr(transparent)]
 pub struct WriteOnly<T: Copy>(T);
 
+impl<T: Debug + Copy> Debug for WriteOnly<T> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> core::fmt::Result {
+        f.debug_tuple("WriteOnly").field(&self.0).finish()
```

Comment:
`WriteOnly` is used to wrap MMIO registers which can only be written, so it doesn't make sense to read a value from it.